src/utils/ar_usd_oracle.ts

```diff
@@ -0,0 +1,65 @@
+import type { FiatOracle } from './fiat_oracle';
+// import CoinGecko from 'coingecko-api';
+import type { CoinGeckoPriceRequestParams, CoinGeckoPriceResponseData, CoinGeckoResponse } from './coingecko_types';
+
+const AR_COIN_ID = 'arweave';
```

Comment:
We might want to make a type that lives alongside the oracle that provides some constants for the currencies, then let the implementation of the oracle map those into whatever shape the oracle's platform expects it for API usage.